src/compiler/utilities.ts

```diff
@@ -249,6 +249,12 @@ namespace ts {
         sourceFile.resolvedTypeReferenceDirectiveNames.set(typeReferenceDirectiveName, resolvedTypeReferenceDirective);
     }
 
+    export function projectReferencesIsEqualTo(oldRef: ProjectReference, newRef: ProjectReference) {

```

Comment:
Consider renaming to `projectReferenceIsEqualTo` since it only handles one reference at a time there's no need for plural in the same